test/rm.js

```diff
@@ -259,27 +280,75 @@ test(
   }
 );
 
+test('remove symbolic link to a file', t => {
+  t.truthy(shell.test('-L', `${t.context.tmp}/link`));
+  const result = shell.rm(`${t.context.tmp}/link`);
+  t.falsy(shell.error());
+  t.is(result.code, 0);
+  t.falsy(shell.test('-L', `${t.context.tmp}/link`));
+  t.falsy(fs.existsSync(`${t.context.tmp}/link`));
+  t.truthy(fs.existsSync(`${t.context.tmp}/file1`));
+});
+
 test('remove symbolic link to a dir', t => {
+  t.truthy(shell.test('-L', `${t.context.tmp}/rm/link_to_a_dir`));
   const result = shell.rm(`${t.context.tmp}/rm/link_to_a_dir`);
   t.falsy(shell.error());
   t.is(result.code, 0);
+  t.falsy(shell.test('-L', `${t.context.tmp}/rm/link_to_a_dir`));
```

Comment:
I think this is already covered by the next line? Or, does `existsSync` follow links? If there's a reason, can you add a comment?